inst/shiny-server/server/ERTPlot.R

```diff
@@ -46,13 +46,14 @@ output$ERTPlot.Multi.Plot <- renderPlotly(
   render_ERTPlot_multi_plot()
 )
 
-render_ERTPlot_multi_plot <- eventReactive(input$ERTPlot.Multi.PlotButton, {
-  req(input$ERTPlot.Multi.Algs)
+render_ERTPlot_multi_plot <- reactive({
+  req(isolate(input$ERTPlot.Multi.Algs))
+  input$ERTPlot.Multi.PlotButton
   withProgress({
   data <- subset(DATA_RAW(),
-                 algId %in% input$ERTPlot.Multi.Algs,
+                 algId %in% isolate(input$ERTPlot.Multi.Algs),
```

Comment:
I suppose `isolate` is the intended behavior, right? We don't want this expression be reactive on input$ERTPlot.Multi.Algs?

Reply:
Yes, this is done in these expensive plots, since a user might be removing algorithms one by one and that would cause too many recalculations of the plot. Instead they can use the refresh-button when they have decided on which algorithms to use.